### ISSUE TITLE
OSD flashing before playback

### ISSUE THREAD

**Issue by tewe** [mpv-player/mpv#2169]
My default is full screen. When I launch the Mac bundle by double-clicking a video, it briefly flashes the "Drop files to play here." message before starting playback. Presumably this happens with other outputs as well. I really don't like it.

**Comment by wm4** [comment]
Duplicate of #1885.
